src/main/java/wis/task/Deadline.java

```diff
@@ -0,0 +1,33 @@
+package wis.task;
+
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.TextStyle;
+import java.util.Locale;
+
+public class Deadline extends Task {
+    LocalDateTime time;
```

Comment:
Perhaps use a more descriptive variable name such as `dueDateTime` as `time` could be a little ambiguous

Reply:
Good suggestion. Noted.